conventions/README.md

```diff
@@ -503,3 +504,11 @@ const elements = this.el.shadowRoot ? this.el.shadowRoot.querySelector("slot").a
 ```
 
 To ensure that all components are compatible for prerendering a prerender build is done as part of `npm test`.
+
+## Writing Tests
+
+### Prevent logging unnecessary messaging in the build
+
+When writing tests, prevent emitting console warnings by stubbing them.
```

Comment:
Could you add something about this only being necessary if a component's test will produce a lot of console messages in a test run?